TACC/Core-Portal
# hotfix: Layout and Appform bugfixes

## Overview: ##
- Temporarily disable removal of falsey parameters, in case of parameter of type bool
- Widen file length cell in data files modals
- Center Onboarding Admin error message

## Testing Steps: ##
1. Open the copy modal on a small file and ensure that `bytes` does not wrap onto new line
2. Go to cep.dev/workbench/onboarding/admin as a non-staff user and confirm message is centered

## UI Photos:

- Before: 
<img width="534" alt="Screen Shot 2021-05-26 at 10 44 04 AM" src="https://user-images.githubusercontent.com/20326896/119690822-a495ce00-be0f-11eb-8fc1-9d4a693e1931.png">

- After: 
<img width="528" alt="Screen Shot 2021-05-26 at 10 28 52 AM" src="https://user-images.githubusercontent.com/20326896/119690814-a19add80-be0f-11eb-8ed9-843f094af4f0.png">
